src/modules/pin-mux/intel-edison-rev-c/intel-edison-rev-c.c

```diff
@@ -399,39 +414,68 @@ static struct mux_pin_map pin_map[] = {
 };
 
 // =============================================================================
+static bool ardu_breakout = false;
+
+static int
+_mux_init(void)
+{
+    //Try to detect the Arduino Breakout
+    if (sol_util_write_file("/sys/class/gpio/export", "214") >= 0) {
```

Comment:
shouldn't we make this another board? The problem is if we can export 214 or not (the board-detect can't write, just read). can't we have another way to do it?